EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/DependenciesPanel.java

```diff
@@ -371,6 +449,46 @@ public void onAfterSubmit(final AjaxRequestTarget target, final Form form) {
         dependencyForm.add(submitButton);
     }
 
+    private void addDeleteValuesButton() {
+        dependencyForm.add(new AjaxLink<Void>("deleteValuesButton") {
+            private static final long serialVersionUID = 1L;
+
+            @Override
+            public void onClick(final AjaxRequestTarget target) {
+                List<DependencyInput> valuesToRemove = new ArrayList<>();
+                if (allSelected && !dependencyValues.isEmpty()) {
+                    valuesToRemove.addAll(dependencyValues);
+                }
+
+                for (DependencyInput dependencyValue : dependencyValues) {
+                    if (dependencyValue.selected) {
+                        valuesToRemove.add(dependencyValue);
+                    }
+                }
+                try {
+                    for (DependencyInput valueToRemove : valuesToRemove) {
+                        AttributeDependency dependency = AttributeDependencyManager.getAttributeDependency(selectedEventType, selectedBaseAttribute, selectedDependentAttribute);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 177 characters long line (which is greater than 150 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS00103)